core/validators/signatures_snapshot.go

```diff
@@ -37,11 +37,17 @@ func (s *ERC20Signatures) SerialisePendingSignatures() *snapshot.ToplogySignatur
 		return pending[i].EthereumAddress < pending[j].EthereumAddress
 	})
 
-	issued := make([]string, 0, len(s.issuedSignatures))
-	for i := range s.issuedSignatures {
-		issued = append(issued, i)
+	issued := make([]*snapshot.IssuedERC20MultisigControlSignature, 0, len(s.issuedSignatures))
+	for resID, data := range s.issuedSignatures {
+		issued = append(issued, &snapshot.IssuedERC20MultisigControlSignature{
+			ResourceId:       resID,
+			EthereumAddress:  data.EthAddress,
+			SubmitterAddress: data.SubmitterAddress,
+		})
 	}
-	sort.Strings(issued)
+	sort.SliceStable(issued, func(i, j int) bool {
+		return issued[i].EthereumAddress < issued[j].EthereumAddress
```

Comment:
I don't think `EthereumAddress` is unique for the issued signature, so it would be best to sort by `resID` instead.